src/imp/tracer_imp.js

```diff
@@ -148,6 +134,21 @@ export default class Tracer extends opentracing.Tracer {
             this.options(opts);
         }
 
+        // For clock skew adjustment.
+        // Must be set after options have been set.
+        this._useClockState = this._options.disable_clock_skew_correction;
```

Comment:
Should this be the opposite?  (i.e. `this._useClockState = !this._options.disable_clock_skew_correction;`)

Reply:
Ah, yes, thanks! I decided to rename the option and forgot to adjust accordingly.